media/js/firefox/new.js

```diff
@@ -27,6 +62,21 @@
                 // (from a download button) then we want them to see the same scene 2
                 // as non-firefox users and initiate a download
                 $html.addClass('firefox-latest');
+
+                uiTourGetConfiguration('appinfo', function(config) {
```

Comment:
Oh yeah, I forgot all the JS for this page is in the `<head>` :cry: 

Given that, doesn't this mean the new code still needs to be in a DOM ready handler to be safe? When this code executes, we can't rely that the markup has been rendered in the tree?